app/controllers/cart_items_controller.rb

```diff
@@ -25,4 +25,13 @@ def show
 
   end
 
+  def destroy
+  item = Item.find(params[:id].to_i)
+   if @cart.contents.delete_if { |item_id,quantity| item_id == params[:id] }
```

Comment:
Unused block argument - `quantity`. If it's necessary, use `_` or `_quantity` as an argument name to indicate that it won't be used.<br>Inconsistent indentation detected.<br>Space missing after comma.